apps/settings/js/panels/wifi_select_certificate_file/panel.js

```diff
@@ -87,18 +84,15 @@ define(function(require) {
               };
 
               certRequest.onerror = function() {
-                // Pop out alert message for certificate import failed
-                var dialog = elements.certificateFailedDialog;
-                dialog.hidden = false;
-                dialog.onsubmit = function confirm() {
-                  dialog.hidden = true;
-                };
-
-                // Re-enable all items of certificate files
-                // since import file process is completed.
-                for (var i = 0; i < items.length; i++) {
-                  items[i].classList.remove('disabled');
-                }
+                DialogService.alert('certificate-import-failed-description', {
+                  title: 'certificate-import-failed'
+                }).then(function() {
+                  // Re-enable all items of certificate files
+                  // since import file process is completed.
+                  for (var i = 0; i < items.length; i++) {
+                    items[i].classList.remove('disabled');
+                  }
```

Comment:
Can we make an inner function for this ? We already did the same thing which tried to add `disabled` class for all items above. So, in order not to duplicate these logics, we can make a function named like `enableCertificateItems(bool)` to integrate these two into !